modules/OCPP/OCPP.cpp

```diff
@@ -116,6 +116,18 @@ static ErrorInfo get_error_info(const std::optional<types::evse_manager::Error>
     }
 }
 
+ocpp::SessionStartedReason get_session_started_reason(const types::evse_manager::StartSessionReason reason) {
+    switch (reason) {
+    case types::evse_manager::StartSessionReason::EVConnected:
+        return ocpp::SessionStartedReason::EVConnected;
+    case types::evse_manager::StartSessionReason::Authorized:
+        return ocpp::SessionStartedReason::Authorized;
+    default:
```

Comment:
@Pietfried best practice here is to do the `throw` outside of the switch.  That way the compiler might see, that some switch cases are missing (for example, if new values are added to the enum).